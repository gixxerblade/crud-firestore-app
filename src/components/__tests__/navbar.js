import React from "react"
import renderer from "react-test-renderer"

import Navbar from "../../components/navbar"

describe("Navbar", ()=>{
    it("Worked better than you thought it would...", ()=>{
        const tree = renderer.create(<Navbar/>).toJSON();
        expect(tree).toMatchSnapshot();
    })
})