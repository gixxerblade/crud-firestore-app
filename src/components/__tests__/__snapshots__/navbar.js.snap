// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Worked better than you thought it would... 1`] = `
Array [
  <hr />,
  <ul
    className="sc-AxjAm cJmUhi"
  >
    <li>
      <a
        aria-current="page"
        href="/"
        onClick={[Function]}
      >
        home
      </a>
    </li>
    <li>
      <a
        href="/about"
        onClick={[Function]}
      >
        about
      </a>
    </li>
    <li>
      <a
        href="/account"
        onClick={[Function]}
      >
        account
      </a>
    </li>
  </ul>,
  <hr />,
]
`;
